src/app/api/geolocation/route.ts

```diff
@@ -1,12 +1,36 @@
+import { SupportedRegions } from '@incubateur-ademe/nosgestesclimat'
+import supportedRegions from '@incubateur-ademe/nosgestesclimat/public/supportedRegions.json' assert { type: 'json' }
 import { NextRequest, NextResponse } from 'next/server'
 import countries from './countries.json'
+import ue_country_codes from './ue_country_codes.json'
 
 export const runtime = 'edge'
 
 export async function GET(request: NextRequest) {
+  const detectedCountryCode = request.geo?.country
+
+  if (detectedCountryCode === null || detectedCountryCode === undefined) {
```

Comment:
Peut-être plus concis ?

```suggestion
  if (!detectedCountryCode) {
```

Reply:
Oui, et ça catchera aussi une string vide